getting-started/assets/cloud_providers/deploy-azure.sh

```diff
@@ -31,6 +33,41 @@ POSTGRES_ADDR=$(echo $CREATE_DB_RESPONSE | jq -r '.host')
 FULL_POSTGRES_ADDR=$(printf '%s\n' "jdbc:postgresql://$POSTGRES_ADDR:5432/{realm}" | sed 's/[&/\]/\\&/g')
 sed -i "/jakarta.persistence.jdbc.url/ s|value=\"[^\"]*\"|value=\"$FULL_POSTGRES_ADDR\"|" "getting-started/assets/eclipselink/persistence.xml"
 
+STORAGE_ACCOUNT_NAME="polaristest$RANDOM_SUFFIX"
+STORAGE_CONTAINER_NAME="polaris-test-container-$RANDOM_SUFFIX"
+
+az storage account create \
+  --name "$STORAGE_ACCOUNT_NAME" \
+  --resource-group "$CURRENT_RESOURCE_GROUP" \
+  --location "$CURRENT_REGION" \
+  --sku Standard_LRS \
+  --kind StorageV2 \
+  --enable-hierarchical-namespace false
+
+az storage container create \
+  --account-name "$STORAGE_ACCOUNT_NAME" \
+  --name "$STORAGE_CONTAINER_NAME" \
+  --auth-mode login
+
+ASSIGNEE_PRINCIPAL_ID=$(az vm show --name $CURRENT_VM_NAME --resource-group $CURRENT_RESOURCE_GROUP --query identity.principalId -o tsv)
+SCOPE=$(az storage account show --name $STORAGE_ACCOUNT_NAME --resource-group $CURRENT_RESOURCE_GROUP --query id -o tsv)
+ROLE="Storage Blob Data Contributor"
+az role assignment create \
+  --assignee $ASSIGNEE_PRINCIPAL_ID \
+  --role "$ROLE" \
+  --scope "$SCOPE"
+
+export AZURE_TENANT_ID=$(az account show --query tenantId -o tsv)
+export STORAGE_LOCATION="abfss://$STORAGE_CONTAINER_NAME@$STORAGE_ACCOUNT_NAME.dfs.core.windows.net/quickstart_catalog"
+
+cat >> getting-started/eclipselink/trino-config/catalog/iceberg.properties << EOF
+fs.native-azure.enabled=true
+azure.auth-type=DEFAULT
+EOF
+
+SPARK_ADDITIONAL_JARS=",org.apache.hadoop:hadoop-azure:3.3.4,org.apache.hadoop:hadoop-azure-datalake:3.3.4,org.apache.iceberg:iceberg-azure-bundle:1.6.1"
+sed -i "/^\s*--packages/s|\",$|${SPARK_ADDITIONAL_JARS}\",|" getting-started/eclipselink/docker-compose.yml
```

Comment:
This is editing the file in-place right? Should we instead make a copy if we need to change the file?

Reply:
Added all JARs to the Docker Compose file directly and tested it. Everything work as expected.